tests/messages.cpp

```diff
@@ -774,12 +774,14 @@ TEST(RoomEvents, CallInvite)
     nlohmann::json data = R"({
           "content": {
             "call_id": "c1591052749788",
+            "party_id": "o912471202012",
             "offer": {
               "sdp": "v=0\r\no=- 6584580628695956864 2 IN IP4 127.0.0.1[...]",
               "type": "offer"
             },
-            "version": 0,
-            "lifetime": 120000
+            "version": 1,
+            "lifetime": 120000,
+            "invitee": "c1591052749784"
           },
           "event_id": "$143273582443PhrSn:example.org",
           "origin_server_ts": 1432735824653,
```

Comment:
Did we decide yet, if we also want to support version 0? If so, we should keep a test for the v0 format.

Reply:
I don't think we have finalised on that yet. Would that be better? I am inclined to keeping support for both the versions.

Reply:
Then please add a separate test to test for version 1 and also verify that it parses the versions appropriately :3